pkg/networkservice/chains/nsmgr/heal_test.go

```diff
@@ -1,424 +0,0 @@
-// Copyright (c) 2020-2021 Doc.ai and/or its affiliates.
-//
-// SPDX-License-Identifier: Apache-2.0
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at:
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package nsmgr_test
```

Comment:
Why do we need to remove heal tests?

Reply:
Because we don't have the heal replacement yet :)  Will revive them when that comes in.